src/sentry/deletions/defaults/group.py

```diff
@@ -259,6 +259,7 @@ def delete_group_hashes(
         finally:
             hash_ids = [gh[0] for gh in hashes_chunk]
             if options.get("deletions.group.delete_group_hashes_metadata_first"):
+                metrics.incr("deletions.group.delete_group_hashes_metadata_first", sample_rate=1.0)
```

Comment:
I want to be certain that this code path is being executed rather than guessing. I will remove it once we're done with this fix.